app/entry.server.tsx

```diff
@@ -1,23 +1,152 @@
-import type { HandleDocumentRequestFunction } from "@remix-run/node";
+import { PassThrough } from "node:stream";
+
+import type { AppLoadContext, EntryContext } from "@remix-run/node";
+import { createReadableStreamFromReadable } from "@remix-run/node";
 import { RemixServer } from "@remix-run/react";
-import { renderToString } from "react-dom/server";
```

Comment:
Single fetch does not work with `renderToString` as @brophdawg11 pointed out. Went ahead and deleted these files and ran `npx remix reveal`. Let me know if there's any issue with that approach

Reply:
Want to just keep them deleted if we don't do anything custom in them?

Reply:
I deleted them in my update - but feel free to add them back if you want them in!